README.md

```diff
@@ -1,519 +1,136 @@
+---
 
-We're releasing it with the community in the spirit of building in the open.
+### Unlock the Power of LLM Evaluation with Lighteval 🚀
 
-Note that it is still very much early so don't expect 100% stability ^^'
-In case of problems or questions, feel free to open an issue!
+**Lighteval** is your all-in-one toolkit for evaluating LLMs across multiple
+backends—whether it's
+[transformers](https://github.com/huggingface/transformers),
+[tgi](https://github.com/huggingface/text-generation-inference),
+[vllm](https://github.com/vllm-project/vllm), or
+[nanotron](https://github.com/huggingface/nanotron)—with
+ease. Dive deep into your model’s performance by saving and exploring detailed,
+sample-by-sample results to debug and see how your models stack-up.
 
-## Installation
+Customization at your fingertips: letting you effortlessly create [new
+tasks](https://github.com/huggingface/lighteval/wiki/Adding-a-Custom-Task) and
+[metrics](https://github.com/huggingface/lighteval/wiki/Adding-a-New-Metric)
+tailored to your needs.
 
-Clone the repo:
+Seamlessly experiment, benchmark, and store your results on the Hugging Face
+Hub, S3, or locally. Empower your AI research with precision and flexibility.
 
-```bash
-git clone https://github.com/huggingface/lighteval.git
-cd lighteval
-```
 
-Create a virtual environment using virtualenv or conda depending on your preferences. We require Python 3.10 or above:
 
-```bash
-conda create -n lighteval python=3.10 && conda activate lighteval
-```
+## 🔑 Key Features
 
-Install the dependencies. For the default installation, you just need:
+- **Speed**: [Use vllm as backend for fast evals](https://github.com/huggingface/lighteval/wiki/Use-VLLM-as-backend).
+- **Seamless Storage**: [Save results in S3 or Hugging Face Datasets](https://github.com/huggingface/lighteval/wiki/Saving-results).
+- **Python API**: [Simple integration with the Python API](https://github.com/huggingface/lighteval/wiki/Use-the-Python-API).
+- **Custom Tasks**: [Easily add custom tasks](https://github.com/huggingface/lighteval/wiki/Adding-a-Custom-Task).
+- **Versatility**: Tons of [metrics](https://github.com/huggingface/lighteval/wiki/Metric-List) and [tasks](https://github.com/huggingface/lighteval/wiki/Available-Tasks) ready to go.
 
-```bash
-pip install .
-```
 
-If you want to evaluate models with frameworks like `accelerate` or `peft`, you will need to specify the optional dependencies group that fits your use case (`accelerate`,`tgi`,`optimum`,`quantization`,`adapters`,`nanotron`,`tensorboardX`):
+## ⚡️ Installation
 
 ```bash
-pip install '.[optional1,optional2]'
+git clone https://github.com/huggingface/lighteval.git
+cd lighteval
+pip install -e .[{extras}]
```

Comment:
Why not a direct and simple pip install?

Reply:
I will add it the next release, for some reason pypi is stucked to the previsous version. I think its because I added `alpha` to the release neame so it automatically marks the release as unstable

Reply:
Might be good to avoid specifying alphas then - having a simpler install in the doc is imo a good reason to do a release, maybe once the docs are good we could do a release, then update, then do the coms?